components/Template.js

```diff
@@ -69,4 +67,44 @@ function transformData(fn, templateKey, originalData) {
   throw new Error('`transformData` must be a function or an object');
 }
 
+function renderTemplate({template, compileOptions, helpers, data}) {
+  var hogan = require('hogan.js');
+  var forOwn = require('lodash/object/forOwn');
+  var content;
+
+  if (typeof template !== 'string' && typeof template !== 'function') {
```

Comment:
Agreed with both of you: `isString`/`isFunction` is much more readable, but this should be amended in another commit.